airbyte-integrations/connectors/source-twilio/source_twilio/manifest.yaml

```diff
@@ -0,0 +1,5416 @@
+        cursor_granularity: PT1S
+        step: P1Y
+      transformations:
+        - type: AddFields
+          fields:
+            - type: AddedFieldDefinition
+              path:
+                - date_created
+              value: '{% set formatted_datetime = format_datetime(record["date_created"], "%Y-%m-%dT%H:%M:%SZ", "%a, %d %b %Y %H:%M:%S %z") %}{{ formatted_datetime }}'
+        - type: AddFields
+          fields:
+            - type: AddedFieldDefinition
+              path:
+                - date_updated
+              value: '{% set formatted_datetime = format_datetime(record["date_updated"], "%Y-%m-%dT%H:%M:%SZ", "%a, %d %b %Y %H:%M:%S %z") %}{{ formatted_datetime }}'
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/Message Media"
+    Usage Records:
+      type: DeclarativeStream
+      name: usage_records
+      primary_key:
+        - account_sid
+        - category
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: >-
+            https://api.twilio.com/2010-04-01/Accounts/{{
+            stream_partition.account_sid }}/Usage/Records.json
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - usage_records
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: Page
+          page_size_option:
+            type: RequestOption
+            field_name: PageSize
+            inject_into: request_parameter
+          pagination_strategy:
+            type: PageIncrement
+            page_size: 1000
+            start_from_page: 0
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: sid
+              partition_field: account_sid
+              stream:
+                $ref: "#/definitions/streams/Accounts"
+      incremental_sync:
+        type: DatetimeBasedCursor
+        cursor_field: start_date
+        cursor_datetime_formats:
+          - "%Y-%m-%dT%H:%M:%SZ"
+        datetime_format: "%Y-%m-%d"
+        start_datetime:
+          type: MinMaxDatetime
+          datetime: '{{ config["start_date"] }}'
+          datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+        start_time_option:
+          type: RequestOption
+          field_name: StartDate>
+          inject_into: request_parameter
+        end_time_option:
+          type: RequestOption
+          field_name: EndDate<
+          inject_into: request_parameter
+        end_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ now_utc().strftime('%Y-%m-%dT%H:%M:%SZ') }}"
+          datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+        cursor_granularity: P1D
```

Comment:
should `cursor_granularity` be 1s in this case because we have datetime format with seconds precision?  doesn't `P1D` lead to missing records?